packages/desktop/src/renderer/components/widgets/userLabel/UserLabel.component.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { styled } from '@mui/material/styles'
+import Grid from '@mui/material/Grid'
+import Typography from '@mui/material/Typography'
+import classNames from 'classnames'
+import WarnIcon from '../../../static/images/warning-icon.svg'
+import { HandleOpenModalType, UserLabelType } from './UserLabel.types'
+
+const PREFIX = 'UserLabel-'
+
+const classes = {
+  messageCard: `${PREFIX}messageCard`,
+  textWhite: `${PREFIX}textWhite`,
+  textBlack: `${PREFIX}textBlack`,
+  wrapper: `${PREFIX}wrapper`,
+  wrapperRed: `${PREFIX}wrapperRed`,
+  wrapperGray: `${PREFIX}wrapperGray`,
+  image: `${PREFIX}image`,
+}
+
+const StyledGrid = styled(Grid)(({ theme }) => ({
+  margin: '-4px 8px 0 4px',
+  [`& .${classes.wrapper}`]: {
+    padding: '1px 6px',
+    borderRadius: '8px',
+    cursor: 'pointer',
+  },
+  [`& .${classes.wrapperGray}`]: {
+    backgroundColor: theme.palette.colors.gray03,
+  },
+  [`& .${classes.wrapperRed}`]: {
+    backgroundColor: theme.palette.colors.error,
+  },
+  [`& .${classes.textWhite}`]: {
+    color: 'white',
+  },
+  [`& .${classes.textBlack}`]: {
+    color: 'black',
+  },
+  [`& .${classes.image}`]: {
+    width: '12px',
+    height: '12px',
+    marginLeft: '4px',
+  },
+}))
+
+export interface UserLabelProps {
+  type: UserLabelType
+  username: string
+  unregisteredUsernameModalHandleOpen?: HandleOpenModalType
+  duplicatedUsernameModalHandleOpen?: HandleOpenModalType
+}
+
+const UserLabel: React.FC<UserLabelProps> = ({
+  type,
+  username,
+  unregisteredUsernameModalHandleOpen,
+  duplicatedUsernameModalHandleOpen,
+}) => {
+  const isUnregistered = type === UserLabelType.UNREGISTERED
+  const handleOpen = isUnregistered ? unregisteredUsernameModalHandleOpen : duplicatedUsernameModalHandleOpen
```

Comment:
How does it look when user is both unregistered and has duplicated username?

Reply:
I built it to provide one label (just like on Figma), we can decide what is more important in that situation and display that label as quick solution.

For long term solution in next iteration we can work on possibility to show 2 different labels